### PR TITLE
[eclipse/xtext#1777] ported xtend code to java

### PR DESCRIPTION
**Pull request by cdietrich**
[eclipse/xtext#1777] ported xtend code to java
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/util/InMemoryURIHandler.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+/**
+ * @since 2.9
+ * 
+ * @noimplement
+ * @noreference
+ * @deprecated Use org.eclipse.xtext.testing.util.InMemoryURIHandler instead
+ */
+@Deprecated
+public class InMemoryURIHandler implements URIHandler {
```
Comment (szarnekow) [inline org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/util/InMemoryURIHandler.java:32]: Out of curiosity: Is it - by chance - possible to say `class InMemoryURIHandler extends ..testing.util.InMemoryURIHandler`?

File: org.eclipse.xtext.xbase.junit/src/org/eclipse/xtext/xbase/junit/formatter/FormatterTester.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.Iterables;
+import com.google.inject.Inject;
+
+/**
+ * @deprecated use org.eclipse.xtext.junit4.formatter.FormatterTester
+ */
+@Deprecated
+public class FormatterTester {
```
Comment (szarnekow) [inline org.eclipse.xtext.xbase.junit/src/org/eclipse/xtext/xbase/junit/formatter/FormatterTester.java:43]: Same question here.

File: org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/util/InMemoryURIHandler.java (hunk 1 above)
Comment (cdietrich) [reply org.eclipse.xtext.junit4/src/org/eclipse/xtext/junit4/util/InMemoryURIHandler.java:32]: afaik there is no relation between the plugins

File: org.eclipse.xtext.xbase.junit/src/org/eclipse/xtext/xbase/junit/formatter/FormatterTester.java (hunk 2 above)
Comment (cdietrich) [reply org.eclipse.xtext.xbase.junit/src/org/eclipse/xtext/xbase/junit/formatter/FormatterTester.java:43]: see above

**Pull request merged**